src/raftstore/store/store.rs

```diff
@@ -1381,6 +1393,9 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             self.apply_worker
                 .schedule(ApplyTask::destroy(job.region_id))
                 .unwrap();
+            self.local_reader
```

Comment:
I think we should unset the remote lease before send the destroy message.

Reply:
+1

Reply:
ping here